bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -400,26 +496,42 @@
 		<label>@text/channel-type.mielecloud.light_switch.label</label>
 		<description>@text/channel-type.mielecloud.light_switch.description</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="light_can_be_controlled">
 		<item-type>Switch</item-type>
 		<label>@text/channel-type.mielecloud.light_can_be_controlled.label</label>
 		<description>@text/channel-type.mielecloud.light_can_be_controlled.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Light</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="plate_power_step">
 		<item-type>String</item-type>
 		<label>@text/channel-type.mielecloud.plate_power_step.label</label>
 		<description>@text/channel-type.mielecloud.plate_power_step.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="plate_power_step_raw">
 		<item-type>Number</item-type>
 		<label>@text/channel-type.mielecloud.plate_power_step_raw.label</label>
 		<description>@text/channel-type.mielecloud.plate_power_step_raw.description</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
```

Comment:
This channel indicates the selected heating level of a plate, so it should rather be `Level` as it has nothing to do with power consumption.